build.sbt

```diff
@@ -30,5 +41,4 @@ lazy val settings = Seq(
     val required = "1.8"
     val current  = sys.props("java.specification.version")
     assert(current == required, s"Unsupported JDK: java.specification.version $current != $required")
-  }
-)
+  }) ++ formattingSettings
```

Comment:
I'm not sure but I wonder if this is applies to the `build.sbt` file because in that case we need the settings to be applied also to root.